### DISCUSSION TITLE
Wrong Bucket size

### DISCUSSION THREAD

**Discussion by piotr-sikora-v** [minio/minio#17613] [Q&A] [answered]
## NOTE
Yesterday I put to one of mu buckets more than 1TB and disk with minio also increase used space more than 1TB so it's correct.
But in min.io console only report 38GB 

## Expected Behavior
show 1.2 TB used space by bucket

## Current Behavior
Show only 38GB

## Steps to Reproduce (for bugs)
I just used `mc mirror`

## Your Environment
minio version RELEASE.2023-07-07T07-13-57Z (commit-id=e20aab25ec26ef3b1e44eac1f6a01f15f043ab42)
Runtime: go1.19.10 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2023 MinIO, Inc.

Docker version.

**Comment by harshavardhana** [comment, marked as answer]
@piotr-sikora-v usage updates happen slowly, observe the speed via `mc admin scanner info alias/`

**Comment by piotr-sikora-v** [comment]
I thought it was happening during PUT.
Thank you!
